### DISCUSSION TITLE
How do I create a simple extension that displays a menu in the sidebar with buttons?

### DISCUSSION THREAD

**Discussion by miaadev** [microsoft/vscode-discussions#1239] [Extension Development QnA] [answered]
I am new to this, and I want to have an item in the activity bar that displays a menu with buttons that can execute code.

**Comment by isidorn** [comment, marked as answer]
Here are a lot of samples https://github.com/microsoft/vscode-extension-samples
You will see that some of them contribute a menu in the package.json
You need to do the same. You can also ask GH Copilot and it can guide you along
